conbench/tests/populate_local_conbench.py

```diff
@@ -275,16 +275,18 @@ def create_benchmarks_data():
 
 
 def create_benchmarks_with_history():
-    # 5 commits in a row in apache/arrow
+    # 7 commits in a row in apache/arrow, one missing
```

Comment:
```suggestion
    # 7 commits in a row in apache/arrow, the commented one is missing
```

Just a bit more clarity on the reason for the commenting?